### PR TITLE
Add UTC+n or UTC-n to the user profile page.

### PR DESCRIPTION
**Pull request by Sysqa**
https://www.pivotaltracker.com/n/projects/982890/stories/78649838

I added this option next to the user timezone that we can see the user timezone differences from UTC

### CODE REVIEW

**Comment by Sysqa** [comment]
@tochman :+1: I also have some modifications (timezone offset column and rake task) on #617 what I hope @christophermlne pull it in that branch soon

File: app/presenters/users/user_presenter.rb [hunk 1]
```diff
@@ -37,6 +37,10 @@ def timezone
     NearestTimeZone.to(user.latitude, user.longitude)
   end
 
+  def timezone_formatted_offset
+    ActiveSupport::TimeZone.new(timezone).formatted_offset
+  end
```
Comment (yggie) [inline app/presenters/users/user_presenter.rb]: `timezone_formatted_offset` seems to be missing tests

**Comment by yggie** [comment]
looks good apart from that missing test :wink:, will merge this in once that is resolved

**Comment by tochman** [comment]
@Sysqa Will you have time to add the test @yggie asked for? I'd love to move ahead with this.

**Comment by Sysqa** [comment]
@tochman yeah, sorry, I'll try to find some time to make this test...

**Comment by tochman** [comment]
Great @Sysqa Thank you! Looking forward to merge this in.

**Comment by tochman** [comment]
Closing and reopening due to Semaphore issues.

**Comment by tochman** [comment]
@Sysqa I'll be pushing code up to staging in a larger PR with code that has accumulated on develop. Will you have time to have a look at this and should I wait for your submission before I start deploying code?

**Pull request merged**